README.md

```diff
@@ -346,8 +346,24 @@ lookup (for physical measurements and biospecimen donation) as well as a
 Participant Work Queue. Any of the above parameters can be provided as a query parameter to do
 an exact match. Examples:
 
-    GET /ParticipantSummary?hpoId=PITT
-    GET /ParticipantSummary?hpoId=PITT&state=PIIState_MA
+    GET /ParticipantSummary?awardee=PITT
+    GET /ParticipantSummary?awardee=PITT&state=PIIState_MA
+
+    GET /ParticipantSummary?organization=PITT_FOO 
+    GET /ParticipantSummary?site=SITE_HPO
+
+The participant summary API supports filtering and sorting on last modified time
+and returns only the participants for the appropriate awardee to a matching Service Account.
```

Comment:
So the way I'm looking at this is filtering and sorting on last_modified is orthogonal to service accounts using this API. Let's remove "and returns...." here.